CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to Stencil
```

Comment:
I don't know that we need a CLA under MIT. We can definitely add the licensing caveat and I'd love to open up the issue queue.